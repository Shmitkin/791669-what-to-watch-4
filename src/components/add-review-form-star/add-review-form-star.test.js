import React from "react";
import renderer from "react-test-renderer";

import AddReviewFormStar from "./add-review-form-star.jsx";

it(`Should AddReviewFormStar render correctly`, () => {
  const tree = renderer
    .create(
        <AddReviewFormStar id={3} checked={true}/>
    )
    .toJSON();

  expect(tree).toMatchSnapshot();
});
