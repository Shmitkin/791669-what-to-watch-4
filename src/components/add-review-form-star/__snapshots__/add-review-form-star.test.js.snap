// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should AddReviewFormStar render correctly 1`] = `
Array [
  <input
    className="rating__input"
    defaultChecked={true}
    disabled={false}
    id="star-3"
    name="rating"
    type="radio"
    value={3}
  />,
  <label
    className="rating__label"
    htmlFor="star-3"
  >
    Rating 
    3
  </label>,
]
`;
